src/widgetastic_patternfly4/chipgroup.py

```diff
@@ -0,0 +1,274 @@
+    """
+
+    CHIP_LOCATOR = (
+        ".//*[(self::li or self::div) and "
+        "contains(@class, 'pf-c-chip') and contains(@class, 'pf-m-overflow')]"
+    )
+
+    @property
+    def is_displayed(self):
+        return self.button.is_displayed
+
+    @property
+    def read_only(self):
+        """
+        Override to always return False here, since the button on this chip is always clickable
+        """
+        return False
+
+    @property
+    def text(self):
+        """
+        Override the text property so this is not a cached property, since button text can change
+        """
+        return self._text.read()
+
+    def _show_less_shown(self):
+        return self.text.replace(" ", "").lower() == "showless"
+
+    def _show_more_shown(self):
+        return "more" in self.text.lower()
+
+    def show_more(self):
+        if self._show_more_shown():
+            self._text.click()
+        wait_for(
+            func=self._show_less_shown,
+            num_sec=3,
+            delay=0.1,
+            message="wait for 'show less' button to appear",
+        )
+
+    def show_less(self):
+        if self._show_less_shown():
+            self._text.click()
+        wait_for(
+            self._show_more_shown,
+            num_sec=3,
+            delay=0.1,
+            message="wait for 'show more' button to appear",
+        )
+
+
+class ChipGroup(View):
+    ROOT = ParametrizedLocator("{@locator}")
+
+    CHIPS_LOCATOR = Chip.CHIP_LOCATOR
+    GROUP_LOCATOR = ".//ul[contains(@class, 'pf-c-chip-group')]"
+    LABEL_LOCATOR = "./preceding-sibling::*[contains(@class, 'pf-c-chip-group__label')]"
+
+    overflow = OverflowChip()
+
+    def __init__(self, parent, locator=None, logger=None):
+        super().__init__(parent, logger=logger)
+        self.locator = locator or self.GROUP_LOCATOR
+
+    @property
+    def label(self):
+        if isinstance(self.parent, ChipGroupToolbarCategory):
+            return self.parent.label
+
+        # It's unlikely we'll have a labelled chip group that is not in a toolbar
+        # ... but just in case
+        elements = self.browser.elements(self.LABEL_LOCATOR)
+        return elements[0].text if elements else None
+
+    def show_more(self):
+        self.overflow.show_more()
+
+    def show_less(self):
+        self.overflow.show_less()
+
+    @property
+    def multiselect(self):
+        return self.overflow.is_displayed
+
+    def get_chips(self, show_more=True):
+        if self.multiselect and show_more:
+            self.show_more()
+        chip_elements = self.browser.elements(self.CHIPS_LOCATOR)
+
+        chips = []
+        for el in chip_elements:
+            # Track the chip by text instead of position, because its position in the row can
+            # change
+            chip_text = quote(self.browser.text(el))
+            chips.append(Chip(parent=self, locator=f"{self.CHIPS_LOCATOR}[(.)={chip_text}]"))
```

Comment:
That's not recommended way of using widgetastic. Widgets shouldn't be instantiated inside methods. Consider using `ParametrizedView` here.

Reply:
I was basing this off my experience with Table in widgetastic.core -- which does instantiate widgets for `TableRow` and `TableColumn` inside of `Table`. I'll search for some ParametrizedView examples

Reply:
Yeah, Table requires refactoring. It was written in very beginning of widgetastic.